package.json

```diff
@@ -27,7 +27,9 @@
   },
   "peerDependencies": {
     "@typescript-eslint/eslint-plugin": "^5.0.0",
-    "@typescript-eslint/parser": "^5.0.0"
+    "@typescript-eslint/parser": "^5.0.0",
+    "eslint": "^8.0.0",
+    "eslint-plugin-import": "^2.0.0"
```

Comment:
It's probably best to match with the versions provided by `eslint-config-airbnb-base`:

https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/package.json#L83-L84

```
"eslint": "^7.32.0 || ^8.2.0",
"eslint-plugin-import": "^2.25.3"
```

Reply:
Derp — you are absolutely right ;)